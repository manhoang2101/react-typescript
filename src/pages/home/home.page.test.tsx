import { shallow } from "enzyme";
import React from "react";
import { render } from "@testing-library/react";
import { HomePage } from ".";

describe("<HomePage />", () => {
  test("toMatchSnapshot()", () => {
    const container = shallow(<HomePage />);
    expect(container).toMatchSnapshot();
  });
  test("renders call", () => {
    const { container } = render(<HomePage />);
    expect(container.textContent).toBe("Home");
  });
});
