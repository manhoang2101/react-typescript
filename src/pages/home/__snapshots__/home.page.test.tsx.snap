// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> toMatchSnapshot() 1`] = `ShallowWrapper {}`;
